test/types/partial/match-param.bad

```diff
@@ -1,10 +1,10 @@
 match-param.chpl:6: In function 'foo':
 match-param.chpl:6: error: unresolved call '==("hello", 100)'
-$CHPL_HOME/modules/internal/String.chpl:408: note: this candidate did not match: ==(x: byteIndex, y: byteIndex)
+$CHPL_HOME/modules/internal/String.chpl:238: note: this candidate did not match: ==(_arg1: chpl__inPlaceBuffer, _arg2: chpl__inPlaceBuffer)
```

Comment:
We use a built-in prediff `diff-ignoring-module-line-numbers` when comparing test output against a .bad file. It irons out internal-module line numbers.

I think this one came up because it is more than just a line-number change.

Reply:
Vass is correct, it came up because the candidate is of different argument types